parity.gemspec

```diff
@@ -9,7 +9,6 @@ Gem::Specification.new do |spec|
   eos
 
   spec.email = ["dan@thoughtbot.com"]
-  spec.executables = ["development", "staging", "production"]
```

Comment:
I was a bit surprised at how this works. The packaging task copies the files from `bin`, and doesn't rely on the gemspec to determine what files to install. The [Homebrew formula](https://github.com/thoughtbot/homebrew-formulae/blob/master/Formula/parity.rb) installs the executables into the Homebrew bin folder.

The `spec.executables` directive tells Rubygems to install those files from `./bin` as executables, but doesn't do anything further.